### ISSUE TITLE
[FEATURE] Psi-Radiation

### ISSUE THREAD

**Issue by Madcombat** [HyperGaming-Network/hgn-srp-server#4]
### Is there an existing request for this?

- [X] I have searched the existing feature

### Explain the feature you would like to request

**What does it do?:** Much alike Radiation, Psi-Radiation would function similiar with the exception that Suits have very little natural resistance to it.   The only piece of equipment that could truely neutrilize Psi-Radiation would be a Psi-helment that would ideally come in afew functions. Ideally, the Psi-radiation Meter would raise much more rapidly and be much more deadly compared to regular radiation, it wouldnt give you a illness but rather simply kill you.   This would be useful for events AND certain regions such as Yantar and Radar.  

**How do you protect against it?:** It'd also be on the expensive side to get calibrated equipment exclusively from the scientists in order to wear one and would likely require becoming a trusted stalker among the scientists as well as doing a bunch of centric scientist jobs in order to actually purchase one.  Ideally the helmets should go for around 40-60K for the Non-Calibrated Variant and the Calibrated variant should go for around 100K+ as it allows actual traversal through places like the Radar or certain underground facilities.

**How would it be useful to events:** Primarily, it can be used as a positive gate keeping device much alike radiation can be used in the same way.    For example: It doesnt make sense for Rookies to raid a secret underground Lab and it's unlikely they'd survive alone based on the conditions.  This outright gate-keeps new characters from performing jobs that only a Veteran Stalker should be prepared to take.    Another way it is useful is for factions like the Monolith, now the Monolith could base in a region that is caked in Psi-radiation and be immune to it which would allow them absolute security in becoming the zones no.1 Nemisis.  This would naturally be extended to Mutants.    

Another way this could be useful is for map traversal, if you want players to avoid certain regions because they're too hazardous, even for someone so well equiped in a suit that protects against the environment, the Psi-radiation would add an extra layer of difficulty when dealing in those regions.

**Any unique visual effects of Psi-Radiation?:** It should apply a yellow grain filter to the players visuals when the count raises, becoming more intense on the way to the max-value.   
![image](https://user-images.githubusercontent.com/20705354/147879897-79895293-c5b4-48da-bf0e-754a810bffe2.png)
Another unique visual would be the audio, you should hear the Psi-emission noise in your characters head.
https://www.youtube.com/watch?v=oJQIiJubWtE&ab_channel=MonolithMaster
_**Fin**_

**Comment by Ireheart** [comment]
Good idea, will add value, but not something to implement until zone expands further and ecologists are introduced
